### PR TITLE
Add mouse coordinate

### LINKED ISSUE #248: Add map coordinates

### PR DESCRIPTION
**Pull request by mbarbet**
Capture d’écran 2020-12-14 à 17 49 54

Fix #248

### CODE REVIEW

**Review by sebbousquet — APPROVED** [review]

File: src/config.json [hunk 1]
```diff
@@ -276,9 +276,9 @@
       }
     },
     "server": {
-      "url": "${ARLAS_SERVER_URL}",
```
Comment (MohamedHamouGisaia) [inline src/config.json:279]: the env variables are removed

File: src/settings.yaml [hunk 2]
```diff
@@ -2,29 +2,29 @@
...
-  require_https: ${ARLAS_AUTHENT_REQUIRE_HTTPS}
-  response_type: ${ARLAS_AUTHENT_RESPONSE_TYPE}
-  silent_refresh_timeout: ${ARLAS_AUTHENT_SILENT_REFRESH_TIMEOUT}
-  timeout_factor: ${ARLAS_AUTHENT_TIMEOUT_FACTOR}
-  session_checks_enabled: ${ARLAS_AUTHENT_ENABLE_SESSION_CHECKS}
-  clear_hash_after_login: ${ARLAS_AUTHENT_CLEAR_HASH}
-  storage: ${ARLAS_AUTHENT_STORAGE}
-  disable_at_hash_check: ${ARLAS_AUTHENT_DISABLE_AT_HASH_CHECK}
-  dummy_client_secret: ${ARLAS_AUTHENT_DUMMY_CLIENT_SECRET}
+  use_authent: false
```
Comment (MohamedHamouGisaia) [inline src/settings.yaml:5]: the env variables are removed

File: src/config.json (hunk 1 above)
Comment (mbarbet) [reply src/config.json]: fixed

Comment (mbarbet) [reply src/config.json]: fixed

File: src/settings.yaml (hunk 2 above)
Comment (mbarbet) [reply src/settings.yaml]: fixed

**Review by MohamedHamouGisaia — APPROVED** [review]

**Pull request merged**